src/App.tsx

```diff
@@ -17,40 +51,18 @@ export const App: React.FC = () => {
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            value={query}
+            onChange={event => {
+              handleQueryChange(event);
+            }}
```

Comment:
```suggestion
            onChange={handleQueryChange}
```
no need to make arrow function. handleQueryChange already except event.